pss/pss.go

```diff
@@ -543,34 +572,28 @@ func (p *Pss) enqueue(msg *PssMsg) error {
 //
 // Will fail if raw messages are disallowed
 func (p *Pss) SendRaw(address PssAddress, topic Topic, msg []byte) error {
+	defer metrics.GetOrRegisterResettingTimer("pss.send.raw", nil).UpdateSince(time.Now())
+
 	if err := validateAddress(address); err != nil {
 		return err
 	}
+
```

Comment:
variables need to be renamed accordingly

Reply:
Which variables? Half of PSS doesn't have appropriate Godoc and I already renamed most of the variables to be consistent with the naming conventions in Go.

This PR is not about fixing and making PSS adhere to all best practices and naming conventions, but about making PSS performant with at least basic workloads of messages, such as those required by Push Sync when uploading a file bigger than 50MB, and adding basic instrumentation so that we can measure various parts of PSS when running under such loads, as the PR description shows.